ci/repo_cleanups/remove_stale_rpm.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash 
+
+
+function run_git {
+#   git for-each-ref --sort=-committerdate refs/remotes --format='%(HEAD)%(refname:short), %(committerdate:relative)'
```

Comment:
As mentioned above I think it's cleaner to remove old code. They're still preserved in git history if it's needed to bring them back later.